cmd/meroxa/turbine/python/internal/cmd.go

```diff
@@ -0,0 +1,41 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type TurbineCommand string
+
+const (
+	TurbineCommandRun     TurbineCommand = "run"
+	TurbineCommandRecord  TurbineCommand = "record"
+	TurbineCommandBuild   TurbineCommand = "build"
+	TurbineCommandVersion TurbineCommand = `version`
+)
+
+func RunTurbineCmd(appPath string, command TurbineCommand, env map[string]string, flags ...string) (string, error) {
+	cmd := NewTurbineCmd(appPath, command, env, flags...)
+	cmd.Stderr = nil
+	cmd.Stdout = nil
+	cmd.Dir = appPath
+	out, err := cmd.CombinedOutput()
+	return strings.TrimSpace(string(out)), err
+}
```

Comment:
I don't think this is used anywhere

Reply:
It is used in version, similarly as we have in Ruby :)